### ISSUE TITLE
clasp/gringo error

### ISSUE THREAD

**Issue by MarioGutierrezR** [hklarner/pyboolnet#99]
Hi, I am new to PyBoolNet so I was trying to run the tutorials, however, there seems to be an issue when calling clasp and/or gringo. For instance, this is the error message I get when running 09_trap_spaces.py:

```
python 09_trap_spaces.py
ERROR call to gringo and / or clasp failed.
ERROR command: /Users/mariogutierrez/miniconda3/lib/python3.9/site-packages/pyboolnet/binaries/gringo-4.4.0/gringo_mac64 | /Users/mariogutierrez/miniconda3/lib/python3.9/site-packages/pyboolnet/binaries/clasp-3.2.0/clasp-3.2.0_mac64 --models=10000 --project --enum-mode=domRec --heuristic=Domain --dom-mod=3,16
ERROR error: *** ERROR: (clasp): parse error in line 0: bad input stream
*** Info : (clasp): Try '--help' for usage information

Traceback (most recent call last):
  File "/Users/mariogutierrez/Documents/red_Booleana/RekaAlbert/pyboolnet/tutorials/09_trap_spaces.py", line 11, in <module>
    mints = compute_trap_spaces(primes, "min")
  File "/Users/mariogutierrez/miniconda3/lib/python3.9/site-packages/pyboolnet/trap_spaces.py", line 245, in compute_trap_spaces
    return potassco_handle(primes, type_, bounds=bounds, project=[], max_output=max_output, fname_asp=fname_asp, representation=representation)
  File "/Users/mariogutierrez/miniconda3/lib/python3.9/site-packages/pyboolnet/external/potassco.py", line 82, in potassco_handle
    raise Exception
Exception
```
PyBoolNet seems to be correctly installed:

```
pip show pyboolnet
Name: pyboolnet
Version: 3.0.9
Summary: Python Toolbox for the generation, manipulation and analysis of Boolean networks.
Home-page: https://github.com/hklarner/pyboolnet
Author: Hannes Klarner
Author-email: hannes.klarner@fu-berlin.de
License: UNKNOWN
Location: /Users/mariogutierrez/miniconda3/lib/python3.9/site-packages
Requires: click, networkx, pytest
Required-by: pystablemotifs
```
But clasp and gringo required to be installed as clingo and not independently. I do not know if this is the problem, though. Would you help us?

Thanks,

Mario

**Comment by hklarner** [comment]
Hello Mario,

this is a clasp parse error:

```
*** ERROR: (clasp): parse error in line 0: bad input stream
```

- can you share `09_trap_spaces.py`?
- can you create the asp file for min trap spaces and share it? (i.e. call `compute_trap_spaces` with an argument for `fname_asp`)
- what version clasp are you using? I have 3.3.4.

**Comment by MarioGutierrezR** [comment]
[09_trap_spaces.py](https://github.com/hklarner/pyboolnet/blob/master/tutorials/09_trap_spaces.py) is the tutorial in this repository.

I'll attach the asp file as .txt.
[file_asp.txt](https://github.com/hklarner/pyboolnet/files/9339672/file_asp.txt)

About clasp:
clasp version 3.3.8
Address model: 64-bit
libclasp version 3.3.8 (libpotassco version 1.1.0)

**Comment by hklarner** [comment]
hm. works for me. Can you try the following Versions?

```
$ clasp --version
clasp version 3.3.4
Address model: 64-bit
libclasp version 3.3.4 (libpotassco version 1.1.0)
Configuration: WITH_THREADS=1
Copyright (C) Benjamin Kaufmann
License: The MIT License <https://opensource.org/licenses/MIT>

$ gringo --version
gringo version 5.4.0
Address model: 64-bit

libgringo version 5.4.0
Configuration: with Python 3.8.2rc2, with Lua 5.3.3
License: The MIT License <https://opensource.org/licenses/MIT>

```

**Comment by MarioGutierrezR** [comment]
Hi, I forgot to post an update regarding this.
At the end I was able to make it work, but I couldn't find the problem.
So, my solution was to change the computer I was working on. I was working from a MacBook so I had to install clasp and gringo from brew as Clingo. However I got the issue that I posted previously, regardless of the version. Thus, I opted to use a WSL, but it also didn't work, although Clasp and Gringo were installed. My final shot was to download python for Windows and surprisingly that was the only way I stopped getting the error and could finally work. 
I must add that I tried environments with different Python and clasp versions in WSL as well as in my Mac, so I am not sure what causes this.
